### ISSUE TITLE
Old link to the bitcoin-dev mailinglist without proper redirection

### ISSUE THREAD

**Issue by raindogdance** [bitcoin-core/bitcoincore.org#1095]
https://bitcoincore.org/en/list/announcements/join/  

The link to the bitcoin-dev mailinglist on the website above doesn't work (returns 'ERROR: No redirect found for https:/lists.linuxfoundation.org/mailman/listinfo/bitcoin-core-dev')

**Comment by kanzure** [comment]
If it is very important that the bitcoin-core-dev old urls get redirected to the announcements that were cross-posted to the bitcoindev mailing lits, then someone could provide me with a mapping of the old-style urls to the new style urls. I will then update my redirection service, which in turn should cause old `lists.linuxfoundation.org` links to work again.

**Comment by laanwj** [comment]
> edit: Actually this is only for the bitcoin-dev mailing list. To my knowledge the announcements list no longer exists. Announcements are usually posted to the bitcoindev mailing list though. See

Correct, there is no announcement list at the moment. Due to the higher and non-specific volume, i think it would be a bad idea to make people subscribe to that for bitcoin core security updates.
